src/main/kotlin/no/nav/familie/ba/sak/config/FeatureToggleConfig.kt

```diff
@@ -95,6 +95,7 @@ class FeatureToggleConfig(private val enabled: Boolean,
         const val TILBAKEKREVING = "familie-ba-sak.behandling.tilbakekreving"
         const val SKJØNNSMESSIGVURDERING = "familie-ba-sak.behandling.skjonnsvurdering"
         const val BRUK_VEDTAKSTYPE_MED_BEGRUNNELSER = "familie-ba-sak.behandling.vedtakstype-med-begrunnelser"
+        const val AUTOMATISK_FØDSELSHENDELSE = "familie-ba-sak.behandling.automatisk-fødselshendelse"
```

Comment:
Tror denne må være uten æøå
```suggestion
        const val AUTOMATISK_FØDSELSHENDELSE = "familie-ba-sak.behandling.automatisk-fodselshendelse"
```
```suggestion
        const val AUTOMATISK_FØDSELSHENDELSE = "familie-ba-sak.behandling.automatisk-fødselshendelse"
```